src/shared/language.ts

```diff
@@ -26,10 +32,12 @@ export const LANGUAGES: Record<string, string> = {
  * @param vscodeLocale - The VSCode locale string to format (e.g., "en-us", "fr-ca")
  * @returns The formatted locale string with uppercase region code
  */
-export function formatLanguage(vscodeLocale: string): string {
+
+export function formatLanguage(vscodeLocale: string): Language {
```

Comment:
@mrubens - Do you think there will be any unintended consequences of strongly typing this return value?

Reply:
I think the language will only work if it's in the list